src/modules/app/components/navigation/navigation.component.scss

```diff
@@ -37,3 +39,7 @@
   cursor: pointer;
   background-color: #dedede;
 }
+
+.custom-nav-list {
```

Comment:
we should add a tailwind-style utility class `overflow-auto` in style.scss to prevent having to name things where we can avoid it

Reply:
I added new classes for overflow and height in style.css